src/app/App.js

```diff
@@ -9,6 +9,8 @@ import FavPage from '../FavPage'
 import GlobalStyles from '../misc/GlobalStyles'
 import DetailPage from '../detailPage/DetailPage'
 import ScrollMemory from 'react-router-scroll-memory'
+import { getMatchedOptions } from '../utils'
+import { getMatchedNum } from '../utils'
 
 let mockPlants = plantObjects
 
```

Comment:
As the App.js file is quite large I recommend to store the optionList in a separate file.